pkgs/top-level/all-packages.nix

```diff
@@ -31196,6 +31196,8 @@ with pkgs;
     inherit (darwin.apple_sdk_11_0.libs) simd;
   };
 
+  dvb_apps  = callPackage ../applications/video/dvb-apps { };
```

Comment:
Please move the package to `pkgs/by-name/dv/dvb-apps/package.nix` and drop this line, thanks.

Reply:
Sounds good. Note that the previous version of the package was named dvb_apps with an underscore. Probably no one will be affected by changing this to a dash since the package has been gone for three years.